lib/src/json_schema_form.dart

```diff
@@ -79,9 +110,10 @@ class _JsonSchemaFormState extends State<JsonSchemaForm> {
                     onPressed: () {
                       if (_formKey.currentState.validate()) {
                         _formKey.currentState.save();
-                        Map<String, dynamic> data = Map<String, dynamic>();
-                        data['submit'] = true;
-                        jsonSchemaBloc.jsonDataAdd.add(data);
+
+                        setState(() {
+                          onFormDataChanged(formData);
+                        });
```

Comment:
2 comments here.

Is setState needed? I don't see any of _JsonSchemaFormState's state that would be mutated in this block.

Does `onFormDataChanged` communicate the use of the callback the best? What do you think of `onSubmit` instead?

Reply:
1. No. This is mistaken, I will remove the `setState`.
2. I think both works fine but feel `onFormDataChanged` is a little more specific what it is expected.